### YamlMime:JavaMember
uid: "com.azure.resourcemanager.containerservice.models.ManagedClusterPodIdentityException.withName*"
fullName: "com.azure.resourcemanager.containerservice.models.ManagedClusterPodIdentityException.withName"
name: "withName"
nameWithType: "ManagedClusterPodIdentityException.withName"
members:
- uid: "com.azure.resourcemanager.containerservice.models.ManagedClusterPodIdentityException.withName(java.lang.String)"
  fullName: "com.azure.resourcemanager.containerservice.models.ManagedClusterPodIdentityException.withName(String name)"
  name: "withName(String name)"
  nameWithType: "ManagedClusterPodIdentityException.withName(String name)"
  summary: "Set the name property: Name of the pod identity exception."
  parameters:
  - description: "the name value to set."
    name: "name"
    type: "<xref href=\"java.lang.String?alt=java.lang.String&text=String\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public ManagedClusterPodIdentityException withName(String name)"
  returns:
    description: "the ManagedClusterPodIdentityException object itself."
    type: "<xref href=\"com.azure.resourcemanager.containerservice.models.ManagedClusterPodIdentityException?alt=com.azure.resourcemanager.containerservice.models.ManagedClusterPodIdentityException&text=ManagedClusterPodIdentityException\" data-throw-if-not-resolved=\"False\" />"
type: "method"
metadata: {}
package: "com.azure.resourcemanager.containerservice.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-containerservice:2.4.0
